canonical-web-and-design/ubuntu.com
# Resolve merge conflict

## Done

- Removed unlinked duplicate cards on /engage
- Removed “======= >>>>>>> Build templates for engage pages from new discourse module”

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/engage
- Check there are no duplicate cards and no unlinked cards

## Issue / Card

None
